packages/carbon/src/classes/Client.ts

```diff
@@ -121,8 +139,11 @@ export class Client {
 	 */
 	constructor(
 		options: ClientOptions,
-		commands: BaseCommand[],
-		plugins: Plugin[] = []
+		handlers: {
+			commands?: BaseCommand[]
+			listeners?: Listener[]
+		},
+		plugins: Plugin[]
```

Comment:
Removed the default value from the plugins parameter so now it is marked as required
```suggestion
		plugins: Plugin[] = []
```